pkg/controller/workloadspread/update_pod_deletion_cost.go

```diff
@@ -83,7 +90,9 @@ func (r *ReconcileWorkloadSpread) syncSubsetPodDeletionCost(
 	replicas := len(activePods)
 
 	// First we partition Pods into two lists: positive, negative list.
-	if subset.MaxReplicas == nil {
+	if subset == nil {
+		negativePods = activePods
```

Comment:
plz comment that we prefer to delete the pods without matching any workloadspread

Reply:
I have added the comments here:
```
func (r *ReconcileWorkloadSpread) updateDeletionCost(ws *appsv1alpha1.WorkloadSpread,
	podMap map[string][]*corev1.Pod,
	workloadReplicas int32) error {
         ... ....
	// update the deletion-cost annotation for such pods that do not match any real subsets.
	// these pods will have the minimum deletion-cost, and will be deleted preferentially.
	if len(podMap[FakeSubsetName]) > 0 {
		if err := r.syncSubsetPodDeletionCost(ws, nil, len(ws.Spec.Subsets), podMap[FakeSubsetName], workloadReplicas); err != nil {
			return err
		}
	}
	return nil
}
```